tinygrad/tensor.py

```diff
@@ -842,6 +842,8 @@ def maximum(self, x:Union[Tensor, Scalar]) -> Tensor: return (self<x).detach().w
   def minimum(self, x:Union[Tensor, Scalar]) -> Tensor: return -((-self).maximum(-x))
 
   def where(self:Tensor, input_:Union[Tensor, Scalar], other:Union[Tensor, Scalar]):
+    if isinstance(input_, Tensor): input_, other = input_._broadcasted(other)
```

Comment:
i think this is fine. for (Tensor, Tensor) we call broadcasted twice (one in return) on them but i don't quite see a better way to write it